src/biotite/sequence/graphics/alignment.py

```diff
@@ -800,7 +935,116 @@ def plot_alignment_type_based(axes, alignment, symbols_per_line=50,
+                                    color=None, cmap=None, fl_score= None,
+                                    color_symbols=False, symbol_spacing=None,
+                                    symbol_size=None, symbol_param=None):
+
+    '''
+    Plots a pairwise sequence alignment using an ArrayPloter instance.
+    Higlights sequence recognition regions at the positions of the respective 
+    score residue per alignment column.
+    
+    Parameters
+    ----------
+    axes : Axes
+        A Matplotlib axes, that is used as plotting area.
+    alignment : Alignment
+        The pairwise sequence alignment to be plotted.
+    symbol_plotter : SymbolPlotter
+        Instance of ArrayPlotter. Defines how the symbols are drawn
+        in the alignment.
+    symbols_per_line : int, optional
+        The amount of alignment columns that are diplayed per line.
+    show_numbers : bool, optional
+        If true, the sequence position of the symbols in the last
+        alignment column of a line is shown on the right side of the
+        plot.
+        If the last symbol is a gap, the position of the last actual
+        symbol before this gap is taken.
+        If the first symbol did not occur up to this point,
+        no number is shown for this line.
+        By default the first symbol of a sequence has the position 1,
+        but this behavior can be changed using the `number_functions`
+        parameter.
+    number_size : float, optional
+        The font size of the position numbers
+    number_functions : list of [(None or Callable(int -> int)], optional
+        By default the position of the first symbol in a sequence is 1,
+        i.e. the sequence position is the sequence index incremented by
+        1.
+        The behavior can be changed with this parameter:
+        If supplied, the length of the list must match the number of
+        sequences in the alignment.
+        Every entry is a function that maps a sequence index (*int*) to
+        a sequence position (*int*) for the respective sequence.
+        A `None` entry means, that the default numbering is applied
+        for the sequence.
+    labels : list of str, optional
+        The sequence labels.
+        Must be the same size and order as the sequences in the
+        alignment.
+    label_size : float, optional
+        Font size of the labels
+    show_line_position : bool, optional
+        If true the position within a line is plotted below the
+        alignment.
+    spacing : float, optional
+        The spacing between the alignment lines. 1.0 means that the size
+        is equal to the size of a symbol box.
+    color : tuple or str, optional
+        A *Matplotlib* compatible color.
+    cmap : Colormap or str, optional
+        The boxes (or symbols, if `color_symbols` is set) are
+        colored based on the normalized intensity value on the
+        given *Matplotlib* Colormap.
+    fl_score : numpy.ndarray 
```

Comment:
Shouldn't this be a mandatory parameter?

Reply:
Yes, you are right. Thanks

Reply:
OK, done.